tools/create_sdk_cipd_unified_package.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+# This script requires depot_tools to be on path.
```

Comment:
could we check that? sanity checks like this are time savers :)

Reply:
We could, but since depot-tools can be installed in any directory, it would not be a robust check.

Reply:
It could look for `which cipd` and exit with code 1 if the string is empty